## Source originally from Fedora - Revision 1.4 of:
http://cvs.fedoraproject.org/viewvc/rpms/vnc/devel/Makefile.am
converted to patch format

--- /dev/null	2008-09-30 20:41:15.000000000 -0700
+++ vnc-4_1_2-unixsrc/unix/xorg-x11-server-source/hw/vnc/Makefile.am	2008-09-30 20:39:33.184092000 -0700
@@ -0,0 +1,60 @@
+LIB_DIR=$(top_srcdir)/../../common
+BIN_DIR=$(top_srcdir)/..
+
+RFB_LIBS=$(LIB_DIR)/rfb/librfb.la
+
+noinst_LTLIBRARIES = libvnccommon.la
+
+libvnccommon_la_SOURCES = vncExtInit.cc vncHooks.cc XserverDesktop.cc
+
+libvnccommon_la_CPPFLAGS = -DVENDOR_RELEASE="$(VENDOR_RELEASE)" \
+	-DVENDOR_STRING="\"$(VENDOR_STRING)\"" -I$(LIB_DIR) \
+	-I$(BIN_DIR)/vncconfig -DGC_HAS_COMPOSITE_CLIP $(XVNC_CPPFLAGS) \
+	-I$(includedir)/pixman-1
+
+bin_PROGRAMS = Xvnc
+
+Xvnc_SOURCES = xvnc.cc $(top_srcdir)/Xext/dpmsstubs.c \
+	$(top_srcdir)/Xi/stubs.c $(top_srcdir)/mi/miinitext.c \
+	$(top_srcdir)/fb/fbcmap_mi.c
+
+nodist_Xvnc_SOURCES = fbrop.h fb.h pixman.h
+
+Xvnc_CPPFLAGS = $(XVNC_CPPFLAGS) -DNO_HW_ONLY_EXTS -DNO_MODULE_EXTS \
+	-DXFree86Server -DVENDOR_RELEASE="$(VENDOR_RELEASE)" \
+	-DVENDOR_STRING="\"$(VENDOR_STRING)\"" -I$(LIB_DIR) \
+	-I$(top_srcdir)/include -I$(includedir)/pixman-1
+
+Xvnc_LDADD = $(XVNC_LIBS) libvnccommon.la $(RFB_LIBS) \
+	$(XSERVER_LIBS) $(XSERVER_SYS_LIBS) -lX11
+
+Xvnc_LDFLAGS = $(LD_EXPORT_SYMBOLS_FLAG)
+
+libvnc_la_LTLIBRARIES = libvnc.la
+libvnc_ladir = $(moduledir)/extensions
+
+libvnc_la_SOURCES = xf86vncModule.cc
+
+libvnc_la_CPPFLAGS = $(XVNC_CPPFLAGS) -I$(LIB_DIR) \
+	-I$(top_srcdir)/hw/xfree86/common \
+	-I$(top_srcdir)/hw/xfree86/os-support \
+	-I$(top_srcdir)/hw/xfree86/os-support/bus \
+	-DXFree86Module -DXFree86LOADER -DIN_MODULE \
+	-I$(includedir)/pixman-1
+
+libvnc_la_LDFLAGS = -module -avoid-version
+
+libvnc_la_LIBADD = libvnccommon.la $(RFB_LIBS)
+
+# C++ hacks
+BUILT_SOURCES = $(nodist_Xvnc_SOURCES)
+
+fb.h: $(top_srcdir)/fb/fb.h
+	cat $(top_srcdir)/fb/fb.h | sed -e 's,and,c_and,' -e 's,xor,c_xor,' > $(srcdir)/fb.h
+
+pixman.h: $(includedir)/pixman-1/pixman.h
+	cat $(includedir)/pixman-1/pixman.h | sed 's/xor/c_xor/' > $(srcdir)/pixman.h
+
+fbrop.h: $(top_srcdir)/fb/fbrop.h
+	cat $(top_srcdir)/fb/fbrop.h | sed -e 's,and,c_and,' -e 's,xor,c_xor,' > $(srcdir)/fbrop.h
+
